### PR TITLE
feat: Degraded Conversation - plain indication

### PR DESCRIPTION
**Pull request by przemvs**
## Description

Displaying system messages for degraded conversation.
There is prepared "base" for displaying messages for other group participants. 

## Screenshots/Screencast (for UI changes)

image

## Checklist

- [x] PR has been self reviewed by the author;
- [x] Hard-to-understand areas of the code have been commented;
- [x] If it is a core feature, unit tests have been added;

### Important details for the reviewers

Other changes will be prepared in another PR to avoid growing this PR.

### CODE REVIEW

File: src/i18n/en-US.json [hunk 1]
```diff
@@ -1335,6 +1335,10 @@
   "tooltipConversationAllDevicesVerified": "All device fingerprints verified (Proteus)",
   "tooltipConversationAllE2EIVerified": "All devices are verified (end-to-end identity). [link]Learn more[/link]",
   "tooltipConversationAllE2EIVerifiedShort": "All devices verified (end-to-end identity)",
+  "tooltipConversationAllE2EINewDeviceAdded": "This conversation is no longer verified, as [bold]{{user}}[/bold] uses at least one device without a valid end-to-end identity certificate.",
+  "tooltipConversationAllE2EINewUserAdded": "This conversation is no longer verified, as [bold]{{user}}[/bold] uses at least one device without a valid end-to-end identity certificate.",
+  "tooltipConversationAllE2EICertificateExpired": "This conversation is no longer verified, as [bold]{{user}}[/bold] uses at least one devices with an expired end-to-end identity certificate.",
+  "tooltipConversationAllE2EICertificateRevoked": "This conversation is no longer verified, as a team admin revoked the end-to-end identity certificate for at least one of [bold]{{user}}[/bold] devices. [link]Learn more[/link]",
```
Comment (PatrykBuniX) [inline src/i18n/en-US.json]: **question**: should it include tooltip in the name? I don't think we render it in a tooltip 🤔

File: src/script/components/MessagesList/Message/E2EIVerificationMessage/E2EIVerificationMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import {E2EIVerificationMessageType} from '../../../../message/E2EIVerificationMessageType';
+
+export interface E2EIVerificationMessageProps {
+  message: E2EIVerificationMessageEntity;
+}
+
+export const E2EIVerificationMessage = ({message}: E2EIVerificationMessageProps) => {
+  const {messageType} = useKoSubscribableChildren(message, ['messageType']);
+
+  const user = 'Deniz Agha';
```
Comment (PatrykBuniX) [inline src/script/components/MessagesList/Message/E2EIVerificationMessage/E2EIVerificationMessage.tsx]: **question**: can't we pass a real user already?

Comment (przemvs) [reply src/script/components/MessagesList/Message/E2EIVerificationMessage/E2EIVerificationMessage.tsx]: I'll do it in another PR :D Because will more things there :)

File: src/style/content/conversation/message-list.less [hunk 3]
```diff
@@ -156,7 +156,7 @@
   &--svg {
     line-height: 0;
 
-    svg:not(.filled) path {
+    svg path {
```
Comment (PatrykBuniX) [inline src/style/content/conversation/message-list.less]: question: Are you sure this won't break anything else? Is this change needed?

Comment (przemvs) [reply src/style/content/conversation/message-list.less]: This one reverted change from previous changes:  https://github.com/wireapp/wire-webapp/pull/15575/commits/394310235b0b194188b316c6e6f9d4813171ed3f#diff-663941c606f2081d8821e5141183537d01ff1a60d7a70ec2b9b38c164f942aefR159

Because I'm changed whole functionality of displaying E2EIVerificationMessage

Comment (PatrykBuniX) [reply src/style/content/conversation/message-list.less]: i see 👌

File: src/i18n/en-US.json (hunk 1 above)
Comment (przemvs) [reply src/i18n/en-US.json]: Updated in this PR: https://github.com/wireapp/wire-webapp/pull/16281/files#diff-ecabae57acd9628cf7ddd155cbcd908b9bdfbffa40be62b46ac42905459ed098

File: src/script/components/MessagesList/Message/E2EIVerificationMessage/E2EIVerificationMessage.test.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+      });
+
+      const {getByTestId} = render(withTheme(<E2EIVerificationMessage message={message} />));
+
+      const elementMessageVerification = getByTestId('element-message-verification');
+      expect(elementMessageVerification.getAttribute('data-uie-value')).toEqual(E2EIVerificationMessageType.VERIFIED);
+    });
+  });
+
+  // TODO: Add more test for another e2ei verification states
```
Comment (przemvs) [inline src/script/components/MessagesList/Message/E2EIVerificationMessage/E2EIVerificationMessage.test.tsx]: More tests will be added after merge this PR: https://github.com/wireapp/wire-webapp/pull/16281

File: .gitignore [hunk 5]
```diff
@@ -1,7 +1,8 @@
 !/.idea/codeStyleSettings.xml
 !/.idea/modules.xml
 .env
-.env.defaults
+.env.*
+!.env.localhost
```
Comment (atomrc) [inline .gitignore]: Those changes should be reverted I think. Doesn't belong here

File: src/script/entity/message/E2EIVerificationMessage.ts [hunk 6]
```diff
@@ -17,21 +17,20 @@
...
-import {SystemMessage} from './SystemMessage';
+import {Message} from './Message';
 
-import {Config} from '../../Config';
-import {SystemMessageType} from '../../message/SystemMessageType';
+import {E2EIVerificationMessageType} from '../../message/E2EIVerificationMessageType';
+import {SuperType} from '../../message/SuperType';
+
+export class E2EIVerificationMessage extends Message {
+  public messageType: ko.Observable<E2EIVerificationMessageType>;
```
Comment (atomrc) [inline src/script/entity/message/E2EIVerificationMessage.ts]: does this need to be observable 🤔 ?

Comment (przemvs) [reply src/script/entity/message/E2EIVerificationMessage.ts]: done :)

File: .gitignore (hunk 5 above)
Comment (przemvs) [reply .gitignore]: reverted ;)

File: src/script/conversation/EventMapper.ts [hunk 7]
```diff
@@ -772,6 +765,16 @@ export class EventMapper {
     return messageEntity;
   }
 
+  /**
+   * Maps JSON data of E2E Identity verification message event to message entity.
+   */
+  private _mapEventE2EIVerificationMessage({data: eventData}: LegacyEventRecord): MissedMessage {
```
Comment (atomrc) [inline src/script/conversation/EventMapper.ts]:
```suggestion
  private _mapEventE2EIVerificationMessage({data: eventData}: E2EIDegradedMessageEvent | AllE2EIVerifiedEvent): MissedMessage {
```

This should work I believe and would provide better typing

**Review by atomrc — APPROVED** [review]
